src/Root.h

```diff
@@ -58,6 +58,8 @@ class cRoot
 
 	void Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo);
 
+	AString m_SettingsFile;
```

Comment:
We usually put variables first, then functions. So move this up.

Also, it's not a "file", it's a "filename".